services/debugger/transformation/transformationStatusUploader.go

```diff
@@ -390,24 +391,6 @@ func (h *Handle) processRecordTransformationStatus(tStatus *TransformationStatus
 					IsError:          true,
 				})
 			}
-		} else if failedEvent.Metadata.MessageID != "" {
```

Comment:
how is this change relevant to the scope of the pull request?

Reply:
not really, but it's the same pattern that we saw in processor, so removed it.